docs/source/es/quicktour.mdx

```diff
@@ -214,7 +214,7 @@ Pasa tu texto al tokenizador:
 El tokenizador devolverá un diccionario conteniendo:
 
 * [input_ids](./glossary#input-ids): representaciones numéricas de los tokens.
-* [atttention_mask](.glossary#attention-mask): indica cuáles tokens deben ser atendidos.
```

Comment:
Good catch! Could you also fix the other 3 occurrences in other `quicktour.mdx` files 🙏, thank you!